README.md

```diff
@@ -1,3 +1,67 @@
+
+Be sure to send us your requested chapter name, which must be short and all lower case. It should ideally be the name of the city where your workshops will be held, or alternatively an area that's small yet meaningful. Choose `san-francisco` or `bay-area` instead of `california` or `usa`.
+
+An existing owner will need to respond and create your chapter repository for you.
+
+Once you are an organizer, follow these instructions to set up the web presence for your ProtoSchool chapter:
+
+**Step 1** | You should have been added to the team called chapter-organizers and should have access to edit the new chapter repo, e.g. https://github.com/protoschool/san-francisco
+
+**Step 2**
+Each chapter can set up their own `gh-pages` branch on their org GitHub pages will automatically route http://proto.school/<reponame> to it.
+
+**Step 3** | You should create a Code of Conduct for your website and repository. You can use this template as a starting point. Be sure to make all people feel welcome at your event.
+
+
+**Step 4** | _This step is important for discoverability!_
+
+Once your chapter is up and running, make a pull request to the ProtoSchool website to add a chapter JSON file to the `/chapters` directory. When your request is accepted, your chapter will be automatically added to the listings at http://proto.school/chapters.html
+
+Your chapter JSON file should look something like this:
+
+```
+{
+  "name": "ProtoSchool San Francisco",
+  "location": "San Francisco, CA",
+  "country": "US",
+  "region": "North America",
+  "organizers": ["mikeal"],
+  "website": "http://proto.school/san-francisco",
+  "twitter": "sfprotoschool",
+  "repo": "http://github.com/protoschool/san-francisco"
+}
+```
+Please be sure to format your JSON data according to the following guidelines. (Fields marked in bold are required.)
+
+| Field | Description |
+| --- | --- |
+| **name** | Whatever you call your events, often <location> ProtoSchool |
+| **location**	 | This appears on the chapters.html page, often <City> or <City, State> but can be any geocode-able string |
+| country	 |  If your location isn't itself a country, use this field to list the ISO-3166 2-letter country code for consistency |
+| **region**  |  Choose an existing region from the chapters page if possible--if not then pick a broad, non-country region name |
+| organizers  | An array of GitHub usernames  |
+| website  |  URL of the main website for your chapter, if one exists besides your GitHub repo |
+| **repo** |  GitHub repo URL |
+| other   | Any other services like Twitter, Gitter, etc. that exist for the chapter (not the organizer's info)  |
+
+
+## Why we take this approach to chapter setup
+- Chapters get their own website.
```

Comment:
"own website at http://proto.school/chapter-name" if they push a gh-pages branch.

we should probably also stick a link to ghpages docs in there.

Reply:
Added that text there but put the link to GH Pages doc up in the setup instructions section.